build.gradle

```diff
@@ -25,21 +25,21 @@ ext {
     slf4jVersion = '1.7.7'
     log4jVersion = '1.2.17'
     jacksonVersion = '2.12.0'
-    branch = System.getenv('TRAVIS_BRANCH') != null
-        ? System.getenv('TRAVIS_BRANCH')
+    branch = System.getenv('ACTIONS_BRANCH') != null
+        ? System.getenv('ACTIONS_BRANCH')
```

Comment:
Similar to Fred's comment. 

Q. Team,  have we decided on a standardized name change for what we had been calling `TRAVIS_BRANCH`??

I have seen us use Both `ACTIONS_BRANCH` (as @alisakotliarova has here) and `BRANCH_NAME`. 

We should pick one version and stick with it.

Thoughts, @FRicci618?